src/features/projects/components/PlantLocation/PlantLocationDetails.tsx

```diff
@@ -146,6 +147,10 @@ export default function PlantLocationDetails({
             )}
             {plantLocation.type === 'single' && <span>{t('1Tree')} </span>}
             {plantLocation.type === 'sample' && <span>{t('sampleTree')} </span>}
+            </div>
+            <div>
+              {plantLocation.hid?plantLocation.hid.substring(0, 3) + "-" + plantLocation.hid.substring(3):null}
```

Comment:
What does HID mean and is it for sure that it has always 3 or more characters (so this code does not fail)?

Reply:
HID is Hex ID. should always be 6 characters.